utils.py

```diff
@@ -311,6 +311,20 @@ def get_current_time_in_millisecs():
     return get_time_in_millisecs(datetime.datetime.utcnow())
 
 
+def get_current_date_as_string():
+    """Returns current date as a string of format 'YYYY-MM-DD'"""
+    return datetime.datetime.utcnow().strftime('%Y-%m-%d')
```

Comment:
Also, add a test for this. Swap out datetime.datetime.utcnow() for your own custom function that returns specific datetimes; check that months, days are zero-padded on the left and that years are four digits long. (Want to make sure that the keys in the storage model are absolutely correct.)